dcos.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Wait for instances to become reachable
-  hosts: all
+  hosts: linux
```

Comment:
to avoid running on windows better use `all!win_agents` but i don't see a reason why we don't want to wait for windows hosts

Reply:
it makes sense, the problem is that Ansible tries to establish ssh connection still